_targets.R

```diff
@@ -19,25 +21,14 @@ source('3_visualize/src/plot_data_coverage.R')
 source('3_visualize/src/map_timeseries.R')
 
 # Configuration
-states <- c('WI','MN','MI', 'IL', 'IN', 'IA')
+states <- c('AL','AZ','AR','CA','CO','CT','DE','DC','FL','GA','ID','IL','IN','IA',
+            'KS','KY','LA','ME','MD','MA','MI','MN','MS','MO','MT','NE','NV','NH',
+            'NJ','NM','NY','NC','ND','OH','OK','OR','PA','RI','SC','SD','TN','TX',
+            'UT','VT','VA','WA','WV','WI','WY','AK','HI','GU','PR')
+#states <- c('WI', 'MN', 'MI')
 parameter <- c('00060')
 
 # Targets
-
-mapped_by_state_targets <- tar_map(
-  values = tibble(state_abb = states) %>%
-    mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
-  tar_target(nwis_inventory, filter(oldest_active_sites, state_cd == state_abb)),
-  tar_target(nwis_data, get_site_data(nwis_inventory, state_abb, parameter)),
-  tar_target(tally, tally_site_obs(nwis_data)),
-  tar_target(timeseries_png,
-             plot_site_data(state_plot_files, nwis_data, parameter),
-             format = 'file'),
-  names = state_abb,
-  unlist = FALSE
-)
-
-
 list(
   # Identify oldest sites
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
```

Comment:
I think you may have missed the step where we fix the part where it re-inventories all when just one state changes. Head back to the 3rd task at https://github.com/wdwatkins/ds-pipelines-targets-3/issues/12#issuecomment-904868710

Reply:
Oh wait ... nevermind. I was up a target to high. Disregard.